typescript/radix-sdk/src/clients/provider.ts

```diff
@@ -160,20 +172,15 @@ export class RadixProvider implements AltVM.IProvider<RadixSDKTransaction> {
   // ### QUERY CORE ###
 
   async getMailbox(req: AltVM.ReqGetMailbox): Promise<AltVM.ResGetMailbox> {
-    return this.query.core.getMailbox({ mailbox: req.mailboxAddress });
+    return getMailboxConfig(this.gateway, req.mailboxAddress);
   }
 
   async isMessageDelivered(req: AltVM.ReqIsMessageDelivered): Promise<boolean> {
-    return this.query.core.isMessageDelivered({
-      mailbox: req.mailboxAddress,
-      message_id: req.messageId,
-    });
+    return isMessageDelivered(this.gateway, req.mailboxAddress, req.messageId);
```

Comment:
I see us switching between object and regular params often. what's the reason for the change here?

Reply:
@ltyu let's say we're stabilising the approach and hope to stop going back and forth soon enough 😇